index.html

```diff
@@ -61,6 +61,172 @@ <h2 class="body-text-header">I'm a Software Developer</h2>
+        </div>
+        <div class="project1-bttns">
+          <button class="project1-btns">html</button>
+          <button class="project1-btns">bootstrap</button>
+          <button class="project1-btns">Ruby</button>
+        </div>
+        <button class="project1-btn">See project</button>
+      </div>
+      <div class="projects-primary">
+        <div class="projects-heading1">
+          <h3>Professional art <span>printing Data</span></p>
+        </div>
+        <div class="projects-disc1">
+          <p>A daily selection of privately personalized reads; no accounts of sign-ups required. has been the industry's standard</p>
+        </div>
+        <div class="project1-bttns">
+          <button class="project1-btns">html</button>
+          <button class="project1-btns">bootstrap</button>
+          <button class="project1-btns">Ruby</button>
+        </div>
+        <button class="project1-btn">See project</button>
+      </div>
+      <div class="projects-primary">
+        <div class="projects-heading1">
+          <h3>Professional art <span>printing Data</span></p>
+        </div>
+        <div class="projects-disc1">
+          <p>A daily selection of privately personalized reads; no accounts of sign-ups required. has been the industry's standard</p>
+        </div>
+        <div class="project1-bttns">
+          <button class="project1-btns">html</button>
+          <button class="project1-btns">bootstrap</button>
+          <button class="project1-btns">Ruby</button>
+        </div>
+        <button class="project1-btn">See project</button>
+      </div>
+      <div class="projects-primary">
+        <div class="projects-heading1">
+          <h3>Professional art <span>printing Data</span></p>
+        </div>
+        <div class="projects-disc1">
+          <p>A daily selection of privately personalized reads; no accounts of sign-ups required. has been the industry's standard</p>
+        </div>
+        <div class="project1-bttns">
+          <button class="project1-btns">html</button>
+          <button class="project1-btns">bootstrap</button>
+          <button class="project1-btns">Ruby</button>
+        </div>
+        <button class="project1-btn">See project</button>
+      </div>
+      <div class="projects-primary">
+        <div class="projects-heading1">
+          <h3>Professional art <span>printing Data</span></p>
+        </div>
+        <div class="projects-disc1">
+          <p>A daily selection of privately personalized reads; no accounts of sign-ups required. has been the industry's standard</p>
+        </div>
+        <div class="project1-bttns">
+          <button class="project1-btns">html</button>
+          <button class="project1-btns">bootstrap</button>
+          <button class="project1-btns">Ruby</button>
+        </div>
+        <button class="project1-btn">See project</button>
```

Comment:
- [ ] As a good practice for professionals, as well as for improved web accessibility, including your `button` type is a good practice i.e. `<button type=""></button>` - which in this case is best if the `type` is set to `button`.